wgsl/index.bs

```diff
@@ -3959,6 +3959,43 @@ specify the component type; the component type is inferred from the constructor
   <thead>
     <tr><th>Precondition<th>Conclusion<th>Notes
   </thead>
+  <tr>
+    <td>|e|: mat2x2&lt;f32&gt;<br>
+    <td>`mat2x2<f32>(`|e|`)`: mat2x2&lt;f32&gt;<br>
+        `mat2x2(`|e|`)`: mat2x2&lt;f32&gt;<br>
+    <td rowspan=9>Identity. The result is |e|.
```

Comment:
```suggestion
    <td rowspan=9>Identity type conversion. The result is |e|.
```

Reply:
I was confused at first by "identity constructor for matrix types", which sounds like mat2x2() -> {1,0,0,1}.